src/waveform/renderers/glwaveformrenderer.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <QtGui/qtguiglobal.h> // for QT_NO_OPENGL and QT_OPENGL_ES_2
```

Comment:
```suggestion
// This is where QT_OPENGL_ES_2 may be defined.
#include <QtGui/qtgui-config.h>
```

This more directly gets at what is needed here. I have confirmed it still builds an aarch64 Flatpak.